### DISCUSSION TITLE
Is there a way to change the `/api` base route?

### DISCUSSION THREAD

**Discussion by crotanite** [nuxt/content#1482] [Q&A] [answered]
I would like to use the content module to display news posts and an faq, but I proxy the `/api` route which breaks all of the content.

Is there a way to change this?

**Comment by nozomuikuta** [comment]
It seems to be unchangeable for now because `/api` is hard-coded, for example:

https://github.com/nuxt/content/blob/ec3c61f3ce6dfd79624551bb3ec9bf3e1330ea59/src/module.ts#L261-L278

So, this would be a feature request.

**Comment by crotanite** [comment, marked as answer]
Yeah at the moment `/api` is hard-coded, so for anyone who finds this and faces a similar problem: I figured out that the proxy package I was using allows regex and the following fixed this

```
proxy: {
    '^/api/(?!.*(_content))': {
        target: process.env.API_URL,
        changeOrigin: true,
        rewrite: (path) => path.replace(/^\/api/, '')
    },
},
```
